stablehlo/conversions/linalg/transforms/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ add_mlir_library(StablehloLinalgTransforms
   Core
 
   LINK_LIBS PUBLIC
+  ChloOps
```

Comment:
Thanks! Just wondering if this revelation part of the same experiment in https://github.com/openxla/stablehlo/pull/2387#issuecomment-2155661581


Nit: Comparing with the bazel file, `StablehloBase` also seems to be missing!

Reply:
> Thanks! Just wondering if this revelation part of the same experiment in [#2387 (comment)](https://github.com/openxla/stablehlo/pull/2387#issuecomment-2155661581)

Yes, this is the case.